flink/v1.20/flink/src/main/java/org/apache/iceberg/flink/maintenance/operator/DeleteFilesProcessor.java

```diff
@@ -40,17 +41,17 @@ public class DeleteFilesProcessor extends AbstractStreamOperator<Void>
     implements OneInputStreamOperator<String, Void> {
   private static final Logger LOG = LoggerFactory.getLogger(DeleteFilesProcessor.class);
 
-  private final String taskIndex;
+  private final String tableName;
   private final String taskName;
+  private final int taskIndex;
   private final SupportsBulkOperations io;
-  private final String tableName;
   private final Set<String> filesToDelete = Sets.newHashSet();
   private final int batchSize;
 
   private transient Counter failedCounter;
   private transient Counter succeededCounter;
 
-  public DeleteFilesProcessor(int taskIndex, String taskName, Table table, int batchSize) {
+  public DeleteFilesProcessor(Table table, String taskName, int taskIndex, int batchSize) {
```

Comment:
There are multiple places where we have these method parameters. Since we are in the early stages, I have decided to standardize the order. Starting from the widest going to the narrowest filter